src/ui/public/courier/__tests__/request_queue.js

```diff
@@ -3,82 +3,37 @@ import expect from 'expect.js';
 import sinon from 'sinon';
 
 import { requestQueue } from '../_request_queue';
-import { SearchStrategyProvider } from '../fetch/strategy/search';
 
 describe('Courier Request Queue', function () {
-  let searchStrategy;
-
   beforeEach(ngMock.module('kibana'));
-  beforeEach(ngMock.inject(function (Private) {
-
-    searchStrategy = Private(SearchStrategyProvider);
-  }));
   beforeEach(requestQueue.clear);
   after(requestQueue.clear);
 
   class MockReq {
-    constructor(strategy, startable = true) {
-      this.strategy = strategy;
+    constructor(startable = true) {
       this.source = {};
       this.canStart = sinon.stub().returns(startable);
     }
   }
 
-  describe('#getStartable(strategy)', function () {
-    it('only returns requests that match one of the passed strategies', function () {
-      requestQueue.push(
-        new MockReq(searchStrategy),
-        new MockReq(searchStrategy),
-        new MockReq(searchStrategy)
-      );
-
-      expect(requestQueue.getStartable(searchStrategy)).to.have.length(3);
-    });
-
-    it('returns all requests when no strategy passed', function () {
+  describe('#getStartable()', function () {
+    it('returns all requests when no default startable value is passed in', function () {
```

Comment:
I took that test out since there is no filtering anymore, so this test is a replacement for the old `'returns all requests when no strategy passed'`, but yea, this test is pretty stupid.  See comment above for a more involved discussion, but this test should go.